examples/metallic_systems.jl

```diff
@@ -23,8 +23,7 @@ atoms = [Mg => [[2/3, 1/3, 1/4], [1/3, 2/3, 3/4]]];
 # `kspacing` far too large to give a converged result.
 # These have been selected to obtain a fast execution time.
 
-kspacing = 0.5      # Minimal spacing of k-points,
-                    # in units of wavevectors (inverse Bohrs)
+kspacing = 0.5      # Minimal spacing of k-points, in units of wavevectors (inverse Bohrs)
```

Comment:
Michael is going to kill you ;-)

Reply:
Ah I see the problem. I guess what I wanted to do was
```
kspacing = 0.5      # Minimal spacing of k-points,
##                     in units of wavevectors (inverse Bohrs)
```

Nice catch!